mvrx-mocking/src/main/kotlin/com/airbnb/mvrx/mocking/printer/MavericksMockPrinter.kt

```diff
@@ -204,6 +205,11 @@ internal abstract class MavericksPrintStateBroadcastReceiver : BroadcastReceiver
     fun register(context: Context) {
         check(!isRegistered) { "Already registered" }
         isRegistered = true
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            context.registerReceiver(networkInfoReceiver, filter, Context.RECEIVER_NOT_EXPORTED)
```

Comment:
@danpe thanks for fixing, but this doesn't seem to compile
```
MavericksMockPrinter.kt:209:38 Unresolved reference: networkInfoReceiver
```

Reply:
I think you need to do what `context.registerReceiver(this, IntentFilter(ACTION_COPY_MAVERICKS_STATE))` does below? and that line should be replaced with this new code

Reply:
Thank you my bad, fixed it.

Reply:
Good to know, but since this is the mocking library it shouldn't be running in production. Do you need to change your configuration?

Reply:
It is weird as even if I disable mocking completely it still crashes,
```
        MockableMavericks.initialize(
            mocksEnabled =false
            debugMode = false,
            applicationContext = this,
            //TODO maybe use other dispatchers
            viewModelCoroutineContext = Dispatchers.Default,
            stateStoreCoroutineContext = Dispatchers.Default
        )
```

Still crashes on startup...